powershell-gallery/powershellget-3.x/Microsoft.PowerShell.PSResourceGet/Compress-PSResource.md

```diff
@@ -0,0 +1,61 @@
+---
+external help file: Microsoft.PowerShell.PSResourceGet.dll-Help.xml
+Module Name: Microsoft.PowerShell.PSResourceGet
+ms.custom: 1.0.5
+ms.date: 05/17/2024
+online version: https://learn.microsoft.com/powershell/module/microsoft.powershell.psresourceget/compress-psresource?view=powershellget-3.x&WT.mc_id=ps-gethelp
+schema: 2.0.0
+---
+
+# Compress-PSResource
+
+## SYNOPSIS
+
+Compresses a specified module from the local computer to a Nupkg ready to be signed and published.
+
+## SYNTAX
+
+```
+Compress-PSResource [-Path] <String> [-DestinationPath] <String> [-PassThru] [-WhatIf]
+ [-Confirm] [<CommonsParameters>]
+```
```

Comment:
This obviously was not created using PlatyPS. You are missing the `## PARAMETERS` section of the article where each parameter is documented.

Also, what version of PSResourceGet contains this cmdlet? We need to coordinate these docs with a release.